modules/installation-aws-permissions.adoc

```diff
@@ -58,6 +59,7 @@ cluster, the IAM user requires the following permissions:
 * `ec2:DescribeVpcClassicLinkDnsSupport`
 * `ec2:DescribeVpcEndpoints`
 * `ec2:DescribeVpcs`
+* `ec2:DisassociateAddress`
```

Comment:
Only required if `publicIpv4Pool` is specified in the install-config.

Reply:
Updated these 2, thanks.